### ISSUE TITLE
 updating your readme

### ISSUE THREAD

**Issue by HamaBarhamou** [Matific/broken-down-models#26]
Hello, Thank you for this package. I will probably need to understand your work to work on this [tiket](https://code.djangoproject.com/ticket/28821) in the next few days. I suggest updating your readme file to explain in more detail how to use your project.

**Comment by shaib** [comment]
Hi, thanks for your work on Django and for your interest in this project. You can find more explanations in the project [documentation](https://broken-down-models.readthedocs.io/en/latest/), the readme file was only intended as a brief introduction.

**Comment by HamaBarhamou** [comment]
great thanks for your work

**Comment by kezabelle** [comment]
👋 May I piggy-back this issue to ask you to throw a _link_ to the docs somewhere, if you get the time? Maybe in GitHub's `About` section, if the `README` isn't a good place for it?

**Comment by HamaBarhamou** [comment]
> 👋 May I piggy-back this issue to ask you to throw a _link_ to the docs somewhere, if you get the time? Maybe in GitHub's `About` section, if the `README` isn't a good place for it?

yes indeed, I think it's a good idea. A link in the readme file to the documentation would be nice.
